packages/admin/admin/src/translator/config/useTranslationConfig.tsx

```diff
@@ -0,0 +1,8 @@
+import * as React from "react";
+
+import { TranslationConfig, TranslationConfigContext } from "./TranslationConfigContext";
+
+export function useTranslationConfig(): TranslationConfig {
+    const context = React.useContext(TranslationConfigContext);
+    return context ?? {};
```

Comment:
`enabled` and `translate` shouldn't be defined optional just for the sake of having an empty object as default value. How about having the following default values?

```ts
React.createContext<TranslationConfig>({ enabled: false, translate: () => { /* Noop */ } })
```